components/popover.scss

```diff
@@ -0,0 +1,6 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/popover";
+@import "../scss/icons/icon";
+@import "../scss/fonts/72";
```

Comment:
I don't believe popovers should require either fonts or icons.  The popover is literally just the container that holds overlay content.  Whatever content goes into the popover will bring with it the font and/or icons that might be needed.

Reply:
(does it use an "X" for closing? - that might be an icon)